PowerShellGet/PSModule.psm1

```diff
@@ -5057,7 +5067,7 @@ function New-ScriptFileInfo
 
         if(-not $Version)
         {
-            $Version = [Version]'1.0'
+            $Version = '1.0'
         }

```

Comment:
Good point, I will add the validation.

---
In reply to: [126790169](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r126790169) [](ancestors = 126790169)